packages/plugins/users-permissions/server/controllers/auth.js

```diff
@@ -100,6 +100,10 @@ module.exports = {
     try {
       const user = await getService('providers').connect(provider, ctx.query);
 
+      if (user.blocked === true) {
+        throw new ApplicationError('Your account is blocked!');
+      }
```

Comment:
```suggestion
      if (user.blocked) {
        throw new ForbiddenError('Your account has been blocked by an administrator');
      }
```
(and add the import for ForbiddenError at the top)

Reply:
Thank you @innerdvations , @christiancp100  for suggesting the changes. I have pushed the commit with suggested changes. 
- Changed the error type to **Forbidden**
- Imported it from utils.errors (similar to other imported errors)
- Removed the redundant `user.blocked === true` statement.  

Although I want to ask a question that, will this give any issue if `user.blocked` comes as a truthy value at some case rather than a boolean.

Reply:
I suggested truthy because IMO it's better to err on the side of caution for security things like this -- for example, if by some strange bug in some case `user.blocked = "true"` we'd let them in.

Admittedly we already do it both ways within this same file 🤦